lib/fork.js

```diff
@@ -19,8 +19,16 @@ module.exports = function (args) {
 	var ps = childProcess.fork(babel, args, options);
 
 	var promise = new Promise(function (resolve, reject) {
+		// define backup results, when there are no tests defined
+		var testResults = {
+			stats: {
+				passCount: 0
```

Comment:
Does `ava` already warn if a test file has no tests? (i.e. `passCount === 0`)?

I would think that should be a failure, otherwise putting:
```js
process.exit(0); 
```
at the top of your test makes it pass. (contrived I know).

Should we be warning if the process exits with code `0`, but never sends a `result` message?